src/components/RecoveryWordsInputField.js

```diff
@@ -0,0 +1,177 @@
+/* global AutoComplete */
+/* global MnemonicPhrase */
+/* global AnimationUtils */
+class RecoveryWordsInputField extends Nimiq.Observable {
+    /**
+     *
+     * @param {number} index
+     */
+    constructor(index) {
+        super();
+
+        this._index = index;
+        /** @type {string} */ this._value = '';
+        this.complete = false;
+
+        this.$ = this._createElements();
+        this._setupAutocomplete();
+    }
+
+    /**
+     * @returns {{ element: HTMLElement, input: HTMLInputElement, placeholder: HTMLDivElement }}
+     */
+    _createElements() {
+        const element = document.createElement('div');
+        element.classList.add('recovery-words-input-field');
+
+        const input = document.createElement('input');
+        input.setAttribute('type', 'text');
+        input.setAttribute('autocorrect', 'off');
+        input.setAttribute('autocapitalize', 'none');
+        input.setAttribute('spellcheck', 'false');
+
+        /** */
+        const setPlaceholder = () => {
+            input.placeholder = `${I18n.translatePhrase('recovery-words-input-field-placeholder')}${this._index + 1}`;
+        };
+        I18n.observer.on(I18n.Events.LANGUAGE_CHANGED, setPlaceholder);
+        setPlaceholder();
+
+        input.addEventListener('keydown', this._onKeydown.bind(this));
+        input.addEventListener('blur', this._onBlur.bind(this));
+        /** @type {HTMLInputElement} */
+        this.$input = input;
+
+        const placeholder = document.createElement('div');
+        placeholder.className = 'placeholder';
+        placeholder.textContent = (this._index + 1).toString();
+        this.$placeholder = placeholder;
+
+        element.addEventListener('click', this._showInput.bind(this));
+        element.addEventListener('mouseenter', this._showInput.bind(this));
+        element.addEventListener('mouseleave', this.showPlaceholder.bind(this));
+        element.appendChild(input);
+
+        // return element;
+        // return { $el: element, $input: input, $placeholder };
+        return { element, input, placeholder };
+    }
+
+    _setupAutocomplete() {
+        this.autocomplete = new AutoComplete({
+            selector: this.$input,
+            source: /** @param{string} term @param{function} response */ (term, response) => {
+                term = term.toLowerCase();
+                const list = MnemonicPhrase.DEFAULT_WORDLIST.filter(word => word.startsWith(term));
+                response(list);
+            },
+            onSelect: this._focusNext.bind(this),
+            minChars: 3,
+            delay: 0,
+        });
+    }
+
+    focus() {
+        requestAnimationFrame(() => this.$.input.focus());
```

Comment:
`requestAnimationFrame` executes instantly if the browser has capacity. What we want here is `setTimeout` without a timeout parameter (or `0`).

Reply:
small numbers cause trouble in Firefox (https://stackoverflow.com/questions/20747591/why-doesnt-this-settimeout-based-code-work-in-firefox-with-a-small-timeout-wor) - I'll put a small number there.